### DISCUSSION TITLE
customize `github-actions[bot]` for deployments / issues / etc.

### DISCUSSION THREAD

**Discussion by ndelangen** [community/community#25853] [Actions]
<p>I’d like to create a deployment status from my action.</p>
<p>I got this working!:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/8038ae21d1fcb5dedc2439b51d0a4fc5d383ed3e.png" data-download-href="/uploads/short-url/iiiyhHQjHGWS9bYdZqy6e5IlNXU.png?dl=1" title="" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3070389/181093733-2899752a-a274-4da6-8f66-2a244f835fcf.png" alt="" data-base62-sha1="iiiyhHQjHGWS9bYdZqy6e5IlNXU" width="690" height="138" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/8038ae21d1fcb5dedc2439b51d0a4fc5d383ed3e_2_690x138.png, https://aws1.discourse-cdn.com/github/optimized/2X/8/8038ae21d1fcb5dedc2439b51d0a4fc5d383ed3e_2_1035x207.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/8/8038ae21d1fcb5dedc2439b51d0a4fc5d383ed3e_2_1380x276.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/8038ae21d1fcb5dedc2439b51d0a4fc5d383ed3e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename"></span><span class="informations">1400×280 28.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I’d like to have a custom name show up instead of <code>github-actions[bot]</code></p>
<p>Is this possible?</p>

**Comment by chrispat** [comment, marked as answer]
<p>That is not possible as that shows the identity of the user who created the status.  And the identity for the GITHUB_TOKEN is the actions bot.</p>

**Comment by ndelangen** [comment]
<p>How do I get another token into the action safely?</p>

**Comment by judoole** [comment]
<p><a class="mention" href="/ndelangen">@ndelangen</a> I think this reflects the token used for creating the deployment event that triggered the Github Action. </p>
<p>So following the figure from <a href="https://developer.github.com/v3/repos/deployments/" rel="nofollow noopener">https://developer.github.com/v3/repos/deployments/</a> , it is the token used in the “Tooling” column.</p>
